core/src/main/java/com/linecorp/armeria/server/streaming/ServerSentEvents.java

```diff
@@ -0,0 +1,348 @@
+     *
+     * @param contentStream the {@link Stream} which publishes the objects supposed to send as contents
+     * @param executor the executor which iterates the stream
+     * @param converter the converter which converts published objects into {@link ServerSentEvent}s
+     */
+    public static <T> HttpResponse fromStream(Stream<T> contentStream, Executor executor,
+                                              Function<? super T, ? extends ServerSentEvent> converter) {
+        return fromStream(defaultHttpHeaders, contentStream, HttpHeaders.EMPTY_HEADERS, executor, converter);
+    }
+
+    /**
+     * Creates a new Server-sent Events stream from the specified {@link Stream} and {@code converter}.
+     *
+     * @param headers the HTTP headers supposed to send
+     * @param contentStream the {@link Stream} which publishes the objects supposed to send as contents
+     * @param executor the executor which iterates the stream
+     * @param converter the converter which converts published objects into {@link ServerSentEvent}s
+     */
+    public static <T> HttpResponse fromStream(HttpHeaders headers, Stream<T> contentStream, Executor executor,
+                                              Function<? super T, ? extends ServerSentEvent> converter) {
+        return fromStream(headers, contentStream, HttpHeaders.EMPTY_HEADERS, executor, converter);
+    }
+
+    /**
+     * Creates a new Server-sent Events stream from the specified {@link Stream} and {@code converter}.
+     *
+     * @param headers the HTTP headers supposed to send
+     * @param contentStream the {@link Stream} which publishes the objects supposed to send as contents
+     * @param trailingHeaders the trailing HTTP headers supposed to send
+     * @param executor the executor which iterates the stream
+     * @param converter the converter which converts published objects into {@link ServerSentEvent}s
+     */
+    public static <T> HttpResponse fromStream(HttpHeaders headers, Stream<T> contentStream,
+                                              HttpHeaders trailingHeaders, Executor executor,
+                                              Function<? super T, ? extends ServerSentEvent> converter) {
+        requireNonNull(headers, "headers");
+        requireNonNull(contentStream, "contentStream");
+        requireNonNull(trailingHeaders, "trailingHeaders");
+        requireNonNull(executor, "executor");
+        requireNonNull(converter, "converter");
+        return streamingFrom(contentStream, sanitizeHeaders(headers), trailingHeaders,
+                             o -> toHttpData(converter, o), executor);
+    }
+
+    /**
+     * Creates a new Server-sent Events stream of the specified {@code content}.
+     *
+     * @param content the object supposed to send as contents
+     */
+    public static <T extends ServerSentEvent> HttpResponse fromServerSentEvent(T content) {
```

Comment:
- `T` could be just `ServerSentEvent`?
- How about just `fromEvent` so that we do not repeat `ServerSentEvent` twice? e.g. `ServerSentEvents.fromServerSentEvent()` vs `ServerSentEvents.fromEvent()`.